src/libraries/Common/src/System/Net/Http/aspnetcore/Http3/QPack/HeaderField.cs

```diff
@@ -19,6 +24,20 @@ public HeaderField(byte[] name, byte[] value)
 
         public byte[] Value { get; }
 
-        public int Length => Name.Length + Value.Length;
+        public int Length => GetLength(Name.Length, Value.Length);
+
+        public static int GetLength(int nameLength, int valueLength) => nameLength + valueLength + RfcOverhead;
+
+        public override string ToString()
+        {
+            if (Name != null)
+            {
+                return Encoding.ASCII.GetString(Name) + ": " + Encoding.ASCII.GetString(Value);
+            }
+            else
+            {
+                return "<empty>";
+            }
+        }
```

Comment:
Does this ToString end up only being used for debugging purposes, or is this surfaced somewhere?

Reply:
It's only for debugging.

Reply:
I think dotnet/runtime isn't using HeaderField at all at the moment.